### PR TITLE
Bootstrap icons v1.0.0-alpha2

### PR DESCRIPTION
**Pull request by meetdilip**
Bootstrap icons for use inside Inkscape as symbols. It is currently in alpha stage. Hopefully, if there are no other issues, I will try to update when the stable branch arrives.

### CODE REVIEW

File: Bootstrap/Bootstrap-icons.svg [hunk 1]
```diff
@@ -0,0 +1,4066 @@
...
+        <dc:contributor>
+          <cc:Agent>
+            <dc:title>meet</dc:title>
+          </cc:Agent>
+        </dc:contributor>
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+  <defs
+     id="defs1483" />
```
Comment (Xaviju) [inline Bootstrap/Bootstrap-icons.svg]: This id can be removed

File: Bootstrap/Bootstrap-icons.svg [hunk 2]
```diff
@@ -0,0 +1,4066 @@
...
+  <sodipodi:namedview
```
Comment (Xaviju) [inline Bootstrap/Bootstrap-icons.svg]: This sodipodi tag can be removed

File: Bootstrap/Bootstrap-icons.svg [hunk 3]
```diff
@@ -0,0 +1,4066 @@
...
+     inkscape:cy="50"
+     inkscape:window-x="0"
+     inkscape:window-y="0"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="svg1479" />
+  <symbol
+     id="alarm-fill"
+     viewBox="0 0 20 20">
+    <path
+       fill-rule="evenodd"
```
Comment (Xaviju) [inline Bootstrap/Bootstrap-icons.svg]: Try to remove this property. Sometimes is not required (is a common issue in figma-generated svgs)

File: Bootstrap/Bootstrap-icons.svg [hunk 4]
```diff
@@ -0,0 +1,4066 @@
...
+       d="M7.5 2.5A.5.5 0 018 2h4a.5.5 0 010 1h-1v1.07a7.002 7.002 0 013.537 12.26l.817.816a.5.5 0 01-.708.708l-.924-.925A6.967 6.967 0 0110 18a6.967 6.967 0 01-3.722-1.07l-.924.924a.5.5 0 01-.708-.708l.817-.816A7.002 7.002 0 019 4.07V3H8a.5.5 0 01-.5-.5zM2.86 7.387A2.5 2.5 0 116.387 3.86 8.035 8.035 0 002.86 7.387zM15.5 3c-.753 0-1.429.333-1.887.86a8.035 8.035 0 013.527 3.527A2.5 2.5 0 0015.5 3zm-5 4a.5.5 0 00-1 0v3.882l-1.447 2.894a.5.5 0 10.894.448l1.5-3A.5.5 0 0010.5 11V7z"
+       clip-rule="evenodd"
```
Comment (Xaviju) [inline Bootstrap/Bootstrap-icons.svg]: Try to remove this property. Sometimes is not required (is a common issue in figma-generated svgs)

File: Bootstrap/Bootstrap-icons.svg (hunk 3 above)
Comment (meetdilip) [reply Bootstrap/Bootstrap-icons.svg]: Ok, I will try and update.

**Comment by meetdilip** [comment]
Ok.  I will try and update.

**Comment by Xaviju** [comment]
Why did you close it?? It only needs minor changes!!

**Comment by meetdilip** [comment]
I am not familiar with Github procedures. I tried to add the new file. I thought I need to make another PR. I will do that in a minute. 

Kindly let me know how to edit keeping the open ( for future use ). Thanks.

**Comment by Xaviju** [comment]
Oh, no problem, you can just enter into the same branch you created, edit the file and commit again. A PR can (and usually) have more than one commit.

Since there are many ways to do it, is hard to explain in a comment.

**Pull request closed without merging**